src/OpenLoco/src/Paint/Paint.cpp

```diff
@@ -1570,37 +1572,37 @@ namespace OpenLoco::Paint
             case 0:
                 for (auto* child = routeEntry.children; child != nullptr; child = child->children)
                 {
-                    minMaxBounds.x = std::max(minMaxBounds.x, child->bounds.x);
-                    minMaxBounds.y = std::max(minMaxBounds.y, child->bounds.y);
-                    minMaxBounds.xEnd = std::min(minMaxBounds.xEnd, child->bounds.xEnd);
-                    minMaxBounds.yEnd = std::min(minMaxBounds.yEnd, child->bounds.yEnd);
+                    minMaxBounds.x = std::min(minMaxBounds.x, child->bounds.x);
```

Comment:
i don't know why i thought it would be max for this it doesn't even pass basic principles as to what you would expect.